backend.go

```diff
@@ -71,12 +71,13 @@ func BackendFromURL(path string) (Backend, error) {
 
 	case "dropbox":
 		return NewStorageDropbox(*u), nil
+	case "backblaze":
+		return NewStorageBackblaze(*u)
 
 	case "s3":
 		fallthrough
 	case "s3s":
 		return NewStorageAmazonS3(*u)
-
```

Comment:
Revoked newline arranging in 5cde247c384ba85491ebd2cd4f1d188ba4e290a3
Also removed println on default case 0c63e38f0f41dc20b3c8c8e094316230d4fa2ba5